src/Tizen.Multimedia/AudioManager/SoundEffect.cs

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Specifies the sound effect types.
+    /// </summary>
+    /// <since_tizen> 12 </since_tizen>
+    public enum SoundEffectType
+    {
+        /// <summary>
+        /// Noise suppression for voice call.
+        /// </summary>
+        NoiseSuppression,
+
+        /// <summary>
+        /// Auto Gain Control for normal capturing.
+        /// </summary>
+        AutoGainControl,
+
+        /// <summary>
+        /// Noise suppression + Auto Gain Control.
+        /// </summary>
+        NsWithAgc,
+
+        /// <summary>
+        /// Including reference source.
+        /// </summary>
+        /// <remarks>
+        /// This effect should be used with <see cref="SoundEffectInfo.ReferenceDevice"/>.
+        /// </remarks>
+        ReferenceCopy = 4097, // 0x1001
```

Comment:
you could have written the hex value and drop the comment, the result would be the same - reference will show 4097 and hex value would be seen in the source code        `ReferenceCopy = 0x1001`, 
as I think the API reference would look the same in both cases, from the API point of view it does matter which one it will be written here

Reply:
Usually we use HEX directly. I'll remove decimal value.